package.json

```diff
@@ -15,17 +15,17 @@
     "test": "npm --prefix packages/client-adapters run test-no-coverage && npm --prefix packages/data-models run test-no-coverage && npm --prefix packages/res-client run test-no-coverage && npm --prefix packages/wordlist run test-no-coverage && npm --prefix packages/inflection-tables run test-no-coverage && npm --prefix packages/components run test-no-coverage",
     "test-ci": "npm --prefix packages/data-models run test-no-coverage && npm --prefix packages/res-client run test-no-coverage && npm --prefix packages/wordlist run test-no-coverage && npm --prefix packages/inflection-tables run test-no-coverage && npm --prefix packages/components run test-no-coverage",
     "build-data-models": "npm --prefix packages/data-models run build",
-    "test-data-models": "npm --prefix packages/data-models run test",
+    "test-data-models": "npm --prefix packages/data-models run test-no-coverage",
     "build-res-cleint": "npm --prefix packages/res-client run build",
-    "test-res-client": "npm --prefix packages/res-client run test",
+    "test-res-client": "npm --prefix packages/res-client run test-no-coverage",
     "build-wordlist": "npm --prefix packages/wordlist run build",
-    "test-wordlist": "npm --prefix packages/wordlist run test",
+    "test-wordlist": "npm --prefix packages/wordlist run test-no-coverage",
     "build-client-adapters": "npm --prefix packages/client-adapters run build",
-    "test-client-adapters": "npm --prefix packages/client-adapters run test",
+    "test-client-adapters": "npm --prefix packages/client-adapters run test-no-coverage",
     "build-inflection-tables": "npm --prefix packages/inflection-tables run build",
-    "test-inflection-tables": "npm --prefix packages/inflection-tables run test",
+    "test-inflection-tables": "npm --prefix packages/inflection-tables run test-no-coverage",
     "build-components": "npm --prefix packages/components run build",
-    "test-components": "npm --prefix packages/components run test-ci",
+    "test-components": "npm --prefix packages/components run test-no-coverage",
```

Comment:
I'm wondering why do we run tests without coverage reporting? Coverage reporting may not work in local environment (it does not work for me at least) but I think this can work in Travis environment and is useful there.

Reply:
It was defined this way before
you could see that in `npm run test` has all no-coverage format
It was not my change, I believe that you, @kirlat or @balmas, defined to do them without coverage - I don't know why.

May be there are some reasons for that?

Reply:
I've run a history check and it shows that it was added in  109689fe0f0bd8ba8e19cb534488b784a1de6453 as `"test-data-models": "npm --prefix packages/data-models run test"` (if to take data models as an example) and then it was changed in your commit 9c75783295fd8788a82e798af15760f788e04f5c to `"test-data-models": "npm --prefix packages/data-models run test-no-coverage"`. That's if it tells the truth.

Reply:
If what is me - than it was not by some reason :)
may be I copied from some place and didn't pay attention to it
and then it was reproduced for everyone.

Anyway - if we want to see coveralls - I could easily change it, of course :)

Reply:
I guess we might as well include the coverage